### ISSUE TITLE
Angular CLI - Cannot find module 'meteor/accounts-base'

### ISSUE THREAD

**Issue by Tusabras** [Urigo/Ionic2CLI-Meteor-WhatsApp#171]
Hey guys, I got stuck during the tutorial, I was trying to do it using the Angular-CLI 
 and I popped up with the problem that the system couldn't find the **accounts-base** module

Here is a link with the code (inside the **api/server/main.ts** there is the import line commented )

https://github.com/Tusabras/meteor-angular-cli

I would really appreciate if someone could help out!:)

**Comment by battistafranco** [comment]
I got he same problem! When I try to build the project I got this message:  Cannot find module 'meteor/accounts-base

**Comment by michaelb-01** [comment]
Same problem here, is there a solution yet?

**Comment by revengelesly** [comment]
look in .meteor folder, find packages
do you have a package called "accounts-password"? 

if not, you have to add it: meteor add accounts-password
try to refresh your server. 

Be advice: I am using meteor with react. But it should work for you to.

**Comment by michaelb-01** [comment]
Yes I do have the accounts-password package, this is my packages file:

```
# Meteor packages used by this project, one per line.
# Check this file (and the other files in this directory) into your repository.
#
# 'meteor add' and 'meteor remove' will edit this file for you,
# but you can also edit it by hand.

meteor-base@1.2.0             # Packages every Meteor app needs to have
mobile-experience@1.0.5       # Packages for a great mobile UX
mongo@1.3.0                   # The database Meteor supports right now
reactive-var@1.0.11            # Reactive variable for tracker
tracker@1.1.3                 # Meteor's client-side reactive programming library

standard-minifier-css@1.3.5   # CSS minifier run for production mode
standard-minifier-js@2.2.0    # JS minifier run for production mode
es5-shim@4.6.15                # ECMAScript 5 compatibility for older browsers
ecmascript@0.9.0              # Enable ECMAScript2015+ syntax in app code
shell-server@0.3.0            # Server-side component of the `meteor shell` command

insecure@1.0.7                # Allow all DB writes from clients (for prototyping)
angular-compilers
accounts-base
accounts-ui
accounts-password
anti:fake

```

One thing to note is that i'm using the meteor-client bundler. It's essentially the same as this:

https://github.com/darkbasic/angularcli-meteor

As a hacky workaround i've disabled my import statement for accounts-base but i've declared Accounts like:

`declare var Accounts: any;`

And I still have the functionality of the accounts package, e.g. I can do Accounts.createUser()

**Comment by ken4ward** [comment]
@michaelb-01 please where did you declare `var Accounts: any;`. I am presently running this project and it's returning the same error. Just help me with file you declared it in.

**Comment by mfechner** [comment]
Same problem here.
Any advice?

**Comment by poparotsy** [comment]
declare var Accounts: any; in phone.ts
and/or 
declare module 'meteor/accounts-base'; in src/declarations.d.ts
